spec/report_spec.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'spec_helper'
+
+describe RepoAudit::Report do
+  let(:repo) { double(Github::Mash, { full_name: 'some-user/not-a-real-repo' }) }
+  let(:report) { described_class.new(repo: repo) }
+  let(:checker) { double(run: nil) }
+
+  before do
+    allow(RepoAudit::LicenseChecker).to receive(:new).with(repo).and_return(checker)
+    allow(ENV).to receive(:fetch).with('GH_TOKEN').and_return('dummy-token')
```

Comment:
Possibly. It always feels a bit weird, to me, to supply tokens as command-line arguments, though. Also, using ENV.fetch makes it very simple to blow things up in an easily understandable way. If you want to have the same effect with a token passed to an initialiser, IMO it makes it a (very) little bit harder to explain to the user what they're supposed to do about it.